bridge/engine/EngineMixer.cpp

```diff
@@ -3834,13 +3845,48 @@ void EngineMixer::checkIfRateControlIsNeeded(const uint64_t timestamp)
     _probingVideoStreams = enableBEProbing;
 }
 
-void EngineMixer::runTransportTicks(const uint64_t timestamp)
+template <typename TStream>
+void removeIdleStreams(concurrency::MpmcHashmap32<size_t, TStream*>& streams,
```

Comment:
[minor] I noticed now this is a free function. If it's a free function it should go to anonymous namespace. But as we receive the pointer to `EngineMixer* mixer` perhaps it make more sense to be member of EngineMixer instead. There is no problem in having a private templated member declared in .h file and the definition on cpp as the function will be only used by the compile unit that has the definition